Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -199,6 +212,24 @@ def __init__(self,
         else:
             self.func = MixedL11Norm()
 
+     # setting the dual value for the proximal calculation - important for warmstart
```

Comment:
This comment would make more sense as a docstring. Comments potentially are dangerous and may not be kept up to date.

Reply:
Thanks - will do